### ISSUE TITLE
Feature Request: Add PostgreSQL, Redis or KeyDB,

### ISSUE THREAD

**Issue by scart88** [rails/mrsk#3]
Adding PostgreSQL, Redis or KeyDB would make this gem more complete. 

Many developers are working on side projects and don't need a managed or HA Redis/PostgreSQL database, so adding these features would be enough to have the app shipped. 

### Redis
```
docker run -d --name redis-stack-server -p 6379:6379 redis/redis-stack-server:latest
```

### KeyDB
```
docker run -p 6379:6379 --restart always --name keydb -d eqalpha/keydb keydb-server /etc/keydb/keydb.conf --server-threads 4 --requirepass password --appendonly yes
```

### PostgreSQL
```
docker run --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword -d postgres
```

**Comment by dhh** [comment]
If you're working on a side project, which is just going to be deployed to a single server, you should just use sqlite. I'll add something to make the mount for that easy. But I don't want to get into running stateful services as part of MRSK. Way too many complications. And again, lots of developers might either get away with just sqlite (if they only run on 1 server) or use a managed DB.

**Comment by AxelTheGerman** [comment]
@dhh fair point on sqlite, sometimes I wish I would have started with it instead of PostgreSQL but wasn't really to popular for actual deployment when I started years ago.

I watched the YouTube video with the walkthrough and you did deploy 2 app servers and 1 DB which is great - not sure if it should be part of MRSK or maybe just documentation but backing up that PG or MySQL would be kinda important

**Comment by laptopmutia** [comment]
yeah I just wish I could do everything with one server, my company only provide me with one vm. but the project is big enough to use sqlite.

currently revert back to using capistrano

**Comment by charnould** [comment]
> If you're working on a side project, which is just going to be deployed to a single server, you should just use sqlite. **I'll add something to make the mount for that easy**.

Sorry to reopen an issue, but I'm trying to set up MRSK with a volume for a SQLite DB but I'm struggling.
In Docker, it would be a simple `named volume`, but in MRSK, `Docs` are not so clear between `Using volumes` chapter and `Using accessories` chapter.

Should I just go with:

```
accessories:
  sqlite:
    volumes:
      - /var/lib/sqlite
```

But It doesn't work...
Any clue for `static volume` where no Docker image is involved and that must be long-lived. Thanks.

**Comment by charnould** [comment]
_For people who are struggling like I was._
After some research, try and fail, the answser was not that hard.  
Below my `mrsk.yaml` to use `SQLite` with `MRSK` 

```
...
service: a_service
image: user/oci_image
servers:
  - 111.111.11.111
volumes:
  - 'database:/app/database'
...
```
In plain english: "mount a `database` (= `database:`) and make it available to my app at `app/database`
